CoreAttributeEditorView/src/au/gov/asd/tac/constellation/views/attributeeditor/AttributeEditorPanel.java

```diff
@@ -304,36 +339,64 @@ private TitledPane createHeaderTitledPane(final HeadingType headingType, final S
         heading.textProperty().bind(title);
         heading.setStyle("-fx-font-weight:bold;");
         heading.setFill(Color.web("#e0e0e0"));
-        final ToggleButton showAllToggle = new ToggleButton("Show all");
-        showAllToggle.setAlignment(Pos.CENTER);
-        showAllToggle.setTextAlignment(TextAlignment.CENTER);
-        showAllToggle.setStyle("-fx-background-insets: 0, 0; -fx-padding: 0");
-        showAllToggle.setPrefSize(60, 12);
-        showAllToggle.setPadding(new Insets(5));
-        showAllToggle.setTooltip(new Tooltip("Show hidden attributes"));
-        final String key;
+        
+        final ToggleButton showEmptyToggle = new ToggleButton("Show Empty");
+        showEmptyToggle.setAlignment(Pos.CENTER);
+        showEmptyToggle.setTextAlignment(TextAlignment.CENTER);
+        showEmptyToggle.setStyle("-fx-background-insets: 0, 0; -fx-padding: 0");
+        showEmptyToggle.setPrefSize(80, 12);
+        showEmptyToggle.setPadding(new Insets(5));
+        showEmptyToggle.setTooltip(new Tooltip("Show empty attributes"));
+        final String emptyKey;
         final GraphElementType elementType;
         switch (headingType) {
             case GRAPH:
-                key = AttributePreferenceKey.GRAPH_SHOW_ALL;
+                emptyKey = AttributePreferenceKey.GRAPH_SHOW_EMPTY;
                 elementType = GraphElementType.GRAPH;
                 break;
             case NODE:
-                key = AttributePreferenceKey.NODE_SHOW_ALL;
+                emptyKey = AttributePreferenceKey.NODE_SHOW_EMPTY;
                 elementType = GraphElementType.VERTEX;
                 break;
             case TRANSACTION:
-                key = AttributePreferenceKey.TRANSACTION_SHOW_ALL;
+                emptyKey = AttributePreferenceKey.TRANSACTION_SHOW_EMPTY;
                 elementType = GraphElementType.TRANSACTION;
                 break;
             default:
-                key = "";
+                emptyKey = "";
                 elementType = null;
                 break;
         }
-        showAllToggle.selectedProperty().addListener((ObservableValue<? extends Boolean> observable, Boolean oldValue, Boolean newValue)
-                -> prefs.putBoolean(key, newValue));
-        showAllToggle.setSelected(prefs.getBoolean(key, false));
+        showEmptyToggle.selectedProperty().addListener((ObservableValue<? extends Boolean> observable, Boolean oldValue, Boolean newValue)
```

Comment:
can you either add finals to the lambda parameters or remove the explicit type definitions?